Document IdentityCredentialError (#40986)
mdn/content: files/en-us/web/api/identitycredentialerror/identitycredentialerror/index.md

---
title: "IdentityCredentialError: IdentityCredentialError() constructor"
short-title: IdentityCredentialError()
slug: Web/API/IdentityCredentialError/IdentityCredentialError
page-type: web-api-constructor
browser-compat: api.IdentityCredentialError.IdentityCredentialError
---

{{APIRef("Fetch API")}}{{AvailableInWorkers}}

The **`IdentityCredentialError()`** constructor creates a new {{domxref("IdentityCredentialError")}} object.

## Syntax

```js-nolint
new IdentityCredentialError()
new IdentityCredentialError(message)
new IdentityCredentialError(message, options)
```

### Parameters

- `message`
  - : A description of the error. If not present, the empty string `''` is used.
- `options` {{optional_inline}}
  - : An object that can have the following properties:
    - `error` {{optional_inline}}

>       - : A string. This can be either a known error from the [OAuth 2.0 specified error list](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1) or an arbitrary string.

Reviewer: ```suggestion
      - : A string. This can be either one of the values listed in the [OAuth 2.0 specified error list](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1) or an arbitrary string.
```
